lib/profile/config.rb

```diff
@@ -34,6 +34,12 @@ def hunter_command
             config.hunter_command ||
             File.join(ENV.fetch('flight_ROOT', '/opt/flight/'), 'bin/flight hunter')
         command.split(' ')
+        if !File.exists?(File.join(command[0], command[1]))
```

Comment:
This is assuming that `command` is an array; it isn't, it's a string. The `command.split(' ')` line above doesn't modify `command`, it was returning the value of `command.split(' ')`.